docs/docs/queries/structure.md

```diff
@@ -1,44 +1,207 @@
 ```
+
+Lists all tasks (completed or not) in your vault
+```dataview
+TASK
+```
+
+Renders a Calendar view where each page is represented as a dot on its creation date.
+```dataview
+CALENDAR file.cday
+```
 ~~~
 
-which will list **all files in your vault**. Find more examples [here](../resources/examples.md).
\ No newline at end of file
+!!! info "Read more about the available types and how to use them [here](./query-types.md)."
+
+## Choose your source
+
+Additionally to the Query Types, you have several **Data Commands** available that help you restrict, refine, sort or group your query. One of these query commands is the **FROM** statement, which is a bit special.
+
+!!! info "`FROM` determines which set of notes should be included in your query depending on given [sources](../../reference/sources)."
+
+You can add **zero or one** `FROM` data command to your query, right after your Query Type. You cannot add multiple FROM statements and you cannot add it after other Data Commands.
+
+`FROM` takes [sources](../../reference/sources) as argument and **restricts the pages your query will collect to the given sources**.
+
+~~~
+Lists all pages inside the folder Books and its sub folders
+```dataview
+LIST
+FROM "Books"
+```
+
+Lists all pages that include the tag #status/open or #status/wip
+```dataview
+LIST
+FROM #status/open OR #status/wip
+```
+
+Lists all pages that have either the tag #assignment and are inside folder "30 School" (or its sub folders), or are inside folder "30 School/32 Homeworks" and are linked on the page School Dashboard Current To Dos
+```dataview
+LIST
+FROM (#assignment AND "30 School") OR ("30 School/32 Homeworks" AND outgoing([[School Dashboard Current To Dos]]))
+```
+
+~~~
+
+!!! info "Read more about `FROM` [here](./data-commands.md#from)."
+
+## Filter, sort, group or limit results
+
+Additionally to the Query Types and the **Data command** `FROM` that's explained above, you have several other **Data Commands** available that help you restrict, refine, sort or group your query. 
+
+All data commands except the `FROM` command can be used **multiple times in any order** (as long as they come after the Query Type and `FROM`, if `FROM` is used at all). They'll be excuted in the order they are written.
+
+Available are:
+
+1. **FROM** like explained [above](#choose-your-source).
+2. **WHERE**: Filter notes based on information **inside** notes, the meta data fields.
+3. **SORT**: Sorts your results depending on a field and a direction.
+4. **GROUP BY**: Bundles up several results into one result row.
+5. **LIMIT**: Limits the output of your query to the given (maximum) number.
+6. **FLATTEN**: Splits up one result into multiple results based on a field or calculation.
+
+~~~
+
+Lists all pages that have a metadata field `due` and where `due` is smaller than today
+```dataview
+LIST
+WHERE due AND due < date(today)
+```
+
+Lists the first 10 pages with the newest creation date and time, that have the tag #status/open
```

Comment:
Comma is extraneous

Reply:
We can be more concise at the risk of clarity:

`Lists the 10 most recently created pages in your vault that have the tag status/open`

vs

`Lists pages that contain #status/open, sorts them in decreasing order of creation time, i.e. most recently created first, and limits the result to the top 10`.